bundles/org.openhab.binding.miio/README.md

```diff
@@ -307,7 +315,7 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
 | Channel          | Type    | Description                         |
 |------------------|---------|-------------------------------------|
 | power            | Switch  | Power                               |
-| mode             | String  | Mode                                |
+| humidifierMode   | String  | Mode                                |
```

Comment:
Maybe update the description too? The channel name has more information than the description does...